README.md

```diff
@@ -82,13 +87,14 @@ It may also be worth including additional codecs for common types that are missi
+| Akka support           | [akka-serialization-jackson](https://doc.akka.io/docs/akka/current/serialization-jackson.html)                                                                                                                                                                                                                                                                                                  | requires custom serializer                                                     | used by [akka-remote](https://doc.akka.io/docs/akka/current/serialization.html) internally     | requires custom serializer                                                                       | requires custom serializer                                                                                                        | [akka-kryo](https://github.com/altoo-ag/akka-kryo-serialization)                                                                                             |
+| Compile-time mechanics | nothing happens in compile time; everything based on runtime reflection                                                                                                                                                                                                                                                                                                                         | derives codecs via [Magnolia](https://github.com/softwaremill/magnolia)        | with ScalaPB, generates Scala classes based on \*.proto files                                  | with Avro4s, derives Avro schemas using Magnolia                                                 | derives codecs **without** Magnolia                                                                                               | with akka-kryo, optionally derives codecs in compile time, but otherwise uses reflection in runtime                                                          |
+| Runtime safety         | none, uses reflection                                                                                                                                                                                                                                                                                                                                                                           | encoders and decoders are checked during compile time                          | standard for Scala code                                                                        | standard for Scala code                                                                          | encoders and decoders are checked during compile time                                                                             | depends on whether codecs were derived in compile time (then standard for Scala code), or not (than none)                                                    |
```

Comment:
Actually, I don't think it's possible to create Avro or Protubuff definition that won't serialise, so four serialisers are runtime safe

Reply:
Okay... how would you suggest to rephrase it for Avro/Protobuf and/or Borer/Circe? 🤔

Reply:
Maybe:
* encoders and decoders are created during compilation
* *.proto files are validated before compilation
* Avro schema is created during compilation

Reply:
Done, patched